docs/sources/userguide/dockervolumes.md

```diff
@@ -141,6 +146,14 @@ be deleted.  To delete the volume from disk, you must explicitly call
 `docker rm -v` against the last container with a reference to the volume. This
 allows you to upgrade, or effectively migrate data volumes between containers.
 
+> **Note:** Docker will not warn you when removing a container *without* 
+> providing the `-v` option to delete its volumes. If you remove containers
+> without using the `-v` option, you may end up with "orphaned" volumes; 
+> volumes that are no longer referenced by a container.
+> Orphaned volumes are difficult to get rid of and can take up a large amount
```

Comment:
Let's not use the term `orphaned`, `dangling` seems to make sense since this is what `docker images` filter uses, and also git.